app/src/main/res/layout/sheet_custom_drug_entry.xml

```diff
@@ -34,8 +34,10 @@
     android:layout_height="wrap_content"
     android:layout_alignParentEnd="true"
     android:layout_gravity="center_vertical"
-    android:layout_marginTop="@dimen/spacing_24"
-    android:layout_marginEnd="@dimen/spacing_24"
+    android:layout_marginTop="@dimen/spacing_18"
```

Comment:
These margins seem wrong. Usually, spacing dimens in Material are multiples of 4. 18dp & 10dp seems odd. Can you take a look at design spec again, please?